### PR TITLE
Email notification when events are suggested

### LINKED ISSUE #61: [Event-Create] Email notification when events are shared/proposed

### PR DESCRIPTION
**Pull request by charles-allen**
## ✅ DoD

- [x] All work is complete
- [x] Migrations: NA
- [x] Release notes: NA
- [x] Issues linked: resolves #61 

## 📝 Summary

- Fixed functions emulation
- Send email when new event suggested

## 💉 Testing

1. Run hosting locally (with firestore & functions emulators)
2. Filled in the form
3. Received the email :)

### Example email

Screen Shot 2021-06-15 at 12 21 51

### Config already deployed

Screen Shot 2021-06-15 at 12 36 55

### CODE REVIEW

File: functions/src/events/index.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+import dedent from 'dedent-js'
```
Comment (antonyharfield) [inline functions/src/events/index.ts]: Space at the top of the file? 😱

**Review by antonyharfield — APPROVED** [review]
📮 Send it!

**Review by grindarius — APPROVED** [review]
Nice one again

File: functions/src/events/index.ts (hunk 1 above)
Comment (charles-allen) [reply functions/src/events/index.ts]: How is it possible?! We have a lint rule about this!!

Comment (charles-allen) [reply functions/src/events/index.ts]: @antonyharfield - Seems `standard-with-typescript` was overriding my rule. I've fixed it!

**Pull request merged**